robot/EDAP/tests/browser/eda_settings_affiliations_edit_mode.robot

```diff
@@ -1,24 +1,24 @@
 *** Settings ***
 
-Resource        robot/HEDAP/resources/HEDA.robot
+Resource        robot/EDAP/resources/EDA.robot
 Suite Setup     Open Test Browser
 Suite Teardown  Delete Records and Close Browser
 
 *** Test Cases ***
 
 Validate Edit Mode For Affiliations, Settings
     [tags]  unstable
-    Go To Heda Settings
+    Go To Eda Settings
     Wait Until Element Is visible	//a[contains(text(),'Affiliations')]
     Click Link				//a[contains(text(),'Affiliations')]
 
     #Set focus to the status field
     Click Element                   //div[@class='slds-button-group']//span[contains(text(), 'Edit')]
     Wait Until Element Is visible   //div[@class='slds-button-group']//span[contains(text(), 'Save')]
-    Textfield Value Should Be       //div[@class='slds-tabs--scoped']//input[@type='text']      StatusTest1
+#    Textfield Value Should Be       //div[@class='slds-tabs--scoped']//input[@type='text']      StatusTest1
```

Comment:
Dead code should be removed. We can always get it back from version control.

Reply:
Removed dead code